tests/test_person.py

```diff
@@ -20,28 +21,45 @@ def test_instance_valriables(self):
 
 	@patch('app.storage')	
 	def test_person_not_in_system(self,test_storage):
-		test_storage.people_info.return_value = {12345678 : "marvin kangethe", 98765432 : "john doe"}
-		self.person.id_no = 22345678
+		#test_storage.people_info.return_value = {12345678 : "marvin kangethe", 98765432 : "john doe"}
+
+		Storage.people_info = {1234 : "marvin kangethe", 9876 : "john doe"}
+		
+		
 		self.assertEqual(
-			type(self.person.id_no),
-			int, msg = "ID numbers must have an integer format.")
+			type(self.person.id_no),int)
 		
 		self.assertFalse(
-			bool(test_storage.people_info.return_value.has_key(self.person.id_no)),
-			msg = "Person already in the system.")
+			bool(Storage.people_info.has_key(self.person.id_no)))
```

Comment:
Doesn't really test anything. Similar to saying `x={1:'a',2:'b',3:'c'}` then  assert if 5 is not a key in x.